packages/ui-components/src/__tests__/StrategyPage.test.ts

```
@@ -152,30 +145,23 @@ describe('StrategySection', () => {
 		await waitFor(() => {
 			expect(screen.getByText('Test Strategy')).toBeInTheDocument();
-			expect(screen.getByTestId('plain-description')).toHaveTextContent(
-				'https://example.com/description.md'
-			);
+			expect(screen.getByTestId('markdown-content')).toBeInTheDocument();
 			expect(mockFetch).toHaveBeenCalledWith('https://example.com/description.md');
 		});
 	});
 
-	it('falls back to plain text when markdown fetch fails', async () => {
+	it.only('falls back to plain text when markdown fetch fails', async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)